Assets/SEE/Game/Worlds/PlayerSpawner.cs

```diff
@@ -9,37 +11,108 @@
 namespace SEE.Game.Worlds
 {
     /// <summary>
-    /// Spawns players in multi-player mode.
+    /// Information needed to spawn a player avatar.
     /// </summary>
-    public class PlayerSpawner : MonoBehaviour
+    [Serializable]
+    class SpawnInfo
```

Comment:
Missing visibility modifier.

Reply:
I moved it into class `PlayerSpawner` as a private nested class.